praetorian_cli/ui/aegis/menu.py

```
@@ -373,17 +514,24 @@ def load_agents(self) -> None:
+                # Build agent_lookup for fast client_id -> hostname mapping
+                self.agent_lookup = {}
+                for agent in self.agents:
+                    if agent.client_id and agent.hostname:
+                        self.agent_lookup[agent.client_id] = agent.hostname
+
             if self.verbose or not self.agents:
                 agent_count = len(self.agents)
                 if agent_count > 0:
                     self.console.print(f"[green]✓ Loaded {agent_count} agents[/green]")
                 else:
                     self.console.print(f"[yellow]⚠ No agents found[/yellow]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)